src/Microsoft.Tye.Core/DockerFileServiceBuilder.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Tye
+{
+    public class DockerFileServiceBuilder : ProjectServiceBuilder
```

Comment:
What does this represent? Is this a .NET project with a Dockerfile?

Or is this meant to capture a non-project that has a Dockerfile?

Bascially the use of inheritance seems like a convenient hack.

Reply:
> Or is this meant to capture a non-project that has a Dockerfile?

This one.

It was a very convenient "refactoring".